css/main.css

```diff
@@ -133,3 +135,159 @@ header .menu img {
   bottom: -52px;
   right: -270px;
 }
+
+.works {
+  width: 100%;
+  margin-top: 12rem;
+}
+
+.works_title {
+  padding: 2.5rem;
+}
+
+.works_title .title {
+  text-align: center;
+  font-family: var(--especialFont);
+  font-size: 4rem;
+  line-height: 5.2rem;
+  padding-bottom: 4rem;
+}
+
+.works .multipost {
+  padding: 2.5rem;
+  margin-top: 3rem;
+}
+
+.works .multipost img {
+  width: 100%;
+}
+
+.works .multipost .multipost_title {
+  font-family: var(--especialFont);
+  font-size: 3.6rem;
+  line-height: 4.4rem;
+  margin-top: 1.5rem;
+}
+
+.works .multipost p {
+  line-height: 2.4rem;
+  margin-top: 1rem;
+}
+
+.works .multipost .tags {
+  display: flex;
+  gap: 1.2rem;
+  margin-top: 1rem;
+}
+
+.works .multipost .tags li {
+  border: 1px solid var(--neutral-grey);
+  padding: 0.8rem 1rem;
+  font-size: 1.5rem;
+}
+
+.works .multipost button {
+  background-color: var(--orange);
+  border: none;
+  color: var(--white);
+  padding: 1rem;
+  margin-top: 1rem;
+  font-size: 1.7rem;
+}
+
+.cards {
+  width: 100%;
+}
+
+.cards .card {
+  margin: 0 2.5rem;
+  background-image: url(../images/imgplaceholderback.jpg);
+  background-repeat: no-repeat;
+  background-position: top center;
+  background-size: cover;
+  margin-bottom: 2rem;
+}
+
+.cards .card .card_container {
+  padding: 15px;
+}
+
+.cards .card .card_container .card_title {
+  font-family: var(--especialFont);
+  color: var(--white);
+  font-size: 3.2rem;
+  line-height: 4.4rem;
+  margin-top: 7.2rem;
```

Comment:
Kindly. To make your website more responsive, you should use display:flex and align all information to the bottom.